<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="3">
           <widget class="QPushButton" name="button_sub">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="shortcut">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="button_1">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="shortcut">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="button_add">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="shortcut">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="button_6">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>6</string>
            </property>
            <property name="shortcut">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="button_mul">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>*</string>
            </property>
            <property name="shortcut">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="button_8">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>8</string>
            </property>
            <property name="shortcut">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="button_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>3</string>
            </property>
            <property name="shortcut">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="button_9">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>9</string>
            </property>
            <property name="shortcut">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="button_7">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>7</string>
            </property>
            <property name="shortcut">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="button_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <property name="shortcut">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="button_4">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="shortcut">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="button_5">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="shortcut">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="button_decimal">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>,</string>
            </property>
            <property name="shortcut">
             <string>,</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="button_0">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="shortcut">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="button_equals">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>=</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="button_div">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>/</string>
            </property>
            <property name="shortcut">
             <string>/</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_clear">
          <property name="text">
           <string>C</string>
          </property>
          <property name="shortcut">
           <string>Backspace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_all_clear">
          <property name="text">
           <string>AC</string>
          </property>
          <property name="shortcut">
           <string>Alt+Backspace</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>318</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
